### ISSUE TITLE
As a user, I want to be able to swipe between routes of a certain rock or sector

### ISSUE THREAD

**Issue by ckummer** [YacGroup/yacguide#249]
When I am viewing a route of a certain rock or sector, I want to be able to swipe between the routes at this rock or in this sector. E. g., when swiping to the right, it should move to the previous route, when swiping to the left, it should move to the next route. At the end of the route list, when swiping to the left, it should show the first route of the list, when swiping to the right at the first route of the list, it should show the last route of the route list.

**Comment by paetz** [comment]
I would call that "Schnick-schnack" but it doesn't hurt to have this feature :)

**Comment by paetz** [comment]
https://www.tutorialspoint.com/how-to-handle-right-to-left-and-left-to-right-swipe-gestures-on-android

**Comment by paetz** [comment]
This may lead to problems since newer phones often use left/right swiping for going back in activity history.

**Comment by ckummer** [comment]
I don't think that this will be a problem, as newer phones do also have galleries where left/right swiping is the standard gesture to move between images. For that reasons, customers have to adopt to this situation anyway.
